storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -0,0 +1,32 @@
+[id="persistent-storage-vsphere"]
+= vSphere CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-vsphere
+
+toc::[]
+
+== Overview
+
+{product-title} can provision persistent volumes (PVs) using the Container Storage Interface (CSI) VMware vSphere driver for Virtual Machine Disk (VMDK) volumes.
+
+:FeatureName: vSphere CSI Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
+
+To create CSI-provisioned persistent volumes (PVs) that mount to vSphere storage assets, {product-title} installs the vSphere Driver Operator and the vSphere driver by default in the `openshift-cluster-csi-drivers` namespace.
+
+* *vSphere CSI Driver Operator*: By default, the Operator provides a storage class that you can use to create PVCs. You also have the option to create the vSphere storage class as described in xref:../../storage/persistent_storage/persistent-storage-gce.adoc#persistent-storage-using-gce[Persistent storage using GCE Persistent Disk].
```

Comment:
Small nit: I realize this is picked up from existing CSI docs, but just now noticing that PV/PVC are not initialized consistently. In each individual file (assembly or module), the term should be spelled out on first instance (for example, "persistent volume claim (PVC)") and then initialized only (PVC) for following instances in the same file.